pkg/manager/risingwave_controller_manager_impl.go

```diff
@@ -58,9 +58,53 @@ type risingWaveControllerManagerImpl struct {
 	forceUpdateEnabled bool
 }
 
+func getStandaloneStatusUtil(rw *risingwavev1alpha1.RisingWave, logger logr.Logger, readyReplicas, requiredReplicas int32, statefulSetName string) risingwavev1alpha1.ComponentReplicasStatus {
+	status := risingwavev1alpha1.ComponentReplicasStatus{
+		Running: 0,
+		Target:  0,
+	}
+
+	if ptr.Deref(rw.Spec.EnableStandaloneMode, false) {
+		status.Target = 1
```

Comment:
`Target` should be equal to the value of `.spec.components.standalone.replicas`.